src/libstore/local.mk

```diff
@@ -4,14 +4,48 @@ libstore_NAME = libnixstore
 
 libstore_DIR := $(d)
 
-libstore_SOURCES := $(wildcard $(d)/*.cc)
+libstore_SOURCES := \
```

Comment:
Since the additional IPFS sources are the second config dependent input for libstore I wanted to make a clean solution instead of following `s3-binary-cache-store.cc` (being the first one) with adding a lot of `#if ENABLE` to the source files. That way only sources are compiled and linked if the config requires them to be. Makes the build processes a bit cleaner and easier to debug.